### DISCUSSION TITLE
FreeBSD Port

### DISCUSSION THREAD

**Discussion by stefs12** [BlitterStudio/amiberry#1222] [General]
Hi, 

I created a FreeBSD port based on the generic x86-64 platform. It is not super clean yet and will most likely require a separate fbsd-x86 platform target. My question is, if that would be something you are interested to add upstream? If cleaned up properly of course.

Basically it does not change much, because the OS-X target already exists, and FreeBSD and OS-X are still quite related.

Right now all my tests seem to indicate that there is a stable SDL2 version for FreeBSD.

best 
Stefan

**Comment by midwan** [comment]
@stefs12 Sure, I don't see why not :)
